pkg/server/ca/rotator/rotator.go

```diff
@@ -124,6 +124,12 @@ func (r *Rotator) rotate(ctx context.Context) error {
 	if x509CAErr != nil {
 		atomic.AddUint64(&r.failedRotationNum, 1)
 		r.c.Log.WithError(x509CAErr).Error("Unable to rotate X509 CA")
+		if r.c.Manager.GetCurrentX509CASlot().IsEmpty() {
+			// Preparation of the X509 CA failed, and there is no active X509
+			// authority. We will be unable to store the JWT authority, so we
```

Comment:
Clarified with @amartinezfayo offline. The specific case we're worried about is that the operation of saving the journal entry in the datastore will always fail without a prepared X.509 authority because both X.509 and JWT authorities are saved in the journal entry.

So this case is correct as described in the comment.